main/src/cgeo/geocaching/storage/ContentStorage.java

```diff
@@ -28,6 +28,7 @@
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Is there a reason why you chose "NotNull" instead of the androidx "NonNull"?

Reply:
Good catch. This was not intended (I followed the suggestion of AndroidStudio and didn't look close enough at the results). I corrected it